### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Inappropriate control type for the Table in "Service Principal Creation Status dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1390]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , select the required subscriptions and select "Start" button.
6. In "Service Principal Creation Status dialog", tab to table and observe the control type using inspect..

**Actual Result:**
Control type for table is list.
Narrator table commands such as "Caps+F3", "Caps+F8" are working fine.

**Narrator :**
Tab
Creating Azure Active Directory application, <say-as interpret-as="spell">2</say-as> of <say-as interpret-as="spell">5</say-as>, selected, 

**NVDA:**
list
tenant ID: 72f988bf-86f1-41af-91ab-2d7cd011db47 ===  not selected  1 of 5

**Expected Result:**
Control type should be Table.
Narrator and NVDA should announce as
Table
Creating Azure Active Directory application, <say-as interpret-as="spell">2</say-as> of <say-as interpret-as="spell">5</say-as>, selected, 

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![service principal creatation status-table-not defined](https://user-images.githubusercontent.com/22956130/35682597-398b9c26-0787-11e8-87cb-b8acef76cf9b.jpg)
![service principal creatation status-table-nvda](https://user-images.githubusercontent.com/22956130/35682600-3c8ac6f4-0787-11e8-93fb-04d9f743fb38.jpg)

**Comment by Eskibear** [comment]
duplicated with #1405

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749847
